### PR TITLE
Created example for creating signals

### PR DESCRIPTION
**Pull request by damascene**
https://github.com/wagtail/wagtail/issues/6991

### CODE REVIEW

File: docs/advanced_topics/adding_signals.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+==================
+How to use Signals
+==================
+
```
Comment (allcaps) [inline docs/advanced_topics/adding_signals.rst:4]: Can we start with a line that points the user to the Django documentation on signals?
https://docs.djangoproject.com/en/3.1/topics/signals/

File: docs/advanced_topics/adding_signals.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        print("Hello world!")
+    
+    # Register listeners for each page model class
+    page_published.connect(receiver, sender=BlogPage)
+    
+
+
+`apps.py`:
+===============
+
```
Comment (allcaps) [inline docs/advanced_topics/adding_signals.rst:48]: Signals might be wired in other locations. Using the app ready is just a convention. We should make that clear.

File: docs/advanced_topics/adding_signals.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    
+        def ready(self):
+            import blog.signals  # noqa
+    
+`__init__.py`:
+===============
+
+.. code-block:: python
+
+    default_app_config = 'blog.apps.BlogConfig'
```
Comment (allcaps) [inline docs/advanced_topics/adding_signals.rst:65]: This is not best practice.
https://docs.djangoproject.com/en/3.2/ref/applications/#configuring-applications last p
> Changed in Django 3.2:
In previous versions, a default_app_config variable in the application module was used to identify the default application configuration class.

**Review by allcaps — COMMENTED** [review]
Thank you for bringing up this topic and improving the docs.

Since Signals is a Django concept, we should point to their documentation a bit more.
- It will be up to date
- It will explain better, with more detail

I do like to have a Signals section, but I think this should focus on Wagtail specifics and for the rest point to Django docs.

Also touching this topic is the difference between Wagtail hooks and Signals.

Confusing to a lot of integrators is the difference between signals and hooks.
- Signals should be used on db operations. The also fire when an object is modified via the ORM.
- Hooks are limited, to their scope. For example only fire when a object is saved via the Wagtail admin.

**Comment by damascene** [comment]
@allcaps Thank you for reviewing this. Please do the required changes if you see fit. I'm not very knowledgeable on the subject. I just wanted a way to make signals works on Wagtail after searching and asking for long time. When I got it to work I wanted to share this short How to with the community. So please feel free to add to it.

**Comment by lb-** [comment]
@damascene thanks so much for submitting this based on what you have learnt, it has been some time and no one else has picked up these changes.

As you noted, you may not be confident to prepare this PR to be ready for review.

I will close this for now and may I suggest you write your learnings up as a blog post, this is a great way to be involved in the community, share what you have learned and maybe the blog post can be added to the This week in Wagtail newsletter or to https://github.com/springload/awesome-wagtail or https://docs.wagtail.org/en/stable/advanced_topics/third_party_tutorials.html

I hope this does not deter you from being involved.

**Comment by damascene** [comment]
Thanks. For some reason I didn't see the comments on my PR.

**Comment by lb-** [comment]
Not sure i follow, your response above implied that you saw the review comments and were not able to continue with this PR yourself.

> @allcaps Thank you for reviewing this. Please do the required changes if you see fit. I'm not very knowledgeable on the subject. I just wanted a way to make signals works on Wagtail after searching and asking for long time. When I got it to work I wanted to share this short How to with the community. So please feel free to add to it.

See the comment above from Coen https://github.com/wagtail/wagtail/pull/6992#pullrequestreview-633029394

**Comment by damascene** [comment]
My bad. almost a year have passed and I've acquired some more info about the subject. I might reopen it in the feature or write a blog page as you have suggested. 
Thank you

**Comment by lb-** [comment]
awesome, sometimes a blog post is a great way to start, it gives you a chance to condense your thoughts into something clear and from there  it can be shared and you can get feedback.

Many of the 'how to' sections in the Wagtail documentation started out as blog posts.

**Pull request closed without merging**